Source/Eto.Gtk/Forms/Controls/DropDownHandler.cs

```diff
@@ -35,6 +36,7 @@ protected override void Initialize()
 		{
 			Create();
 			container = new Gtk.EventBox();
+			container.WidthRequest = 150;
```

Comment:
Kind of, if it's placed inside a layout it will expand if the layout requests it to expand, but it will not try to set it's own minimum size to the longest item size.